src/styles/blocks/card.scss

```
@@ -0,0 +1,117 @@
+@import "./variables.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin-right: $gap-between-cards;
+  margin-bottom: 46px;
+  justify-content: center;
+  width: $product-card-width;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: flex;
+    height: 134px;
+    width: 160px;
+    margin: 32px 19px auto;
+  }
+
+  &__model {
+    display: flex;
+    color: #060b35;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin: 40px 16px 0;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-top: 4px;
+    margin-left: 16px;
+    margin-bottom: 0;
+  }
+
+  &__review {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: center;
+    align-items: center;
+    margin: 16px;
+    margin-bottom: 0;
+
+    &_number {
+      display: flex;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: #060b35;
+      margin: 0;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 16px 0;
+
+    &-text {
+      display: inline-block;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+      margin: 0;
+    }
+
+    &-value {
+      display: flex;
+      justify-content: flex-end;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: #060b35;
+      margin: 0;
+    }
+  }
+
+  &__button {
```

```suggestion
  &__button {
  box-sizing: border-box;
```
When you hover on button, all catalog moving 2px according to border width